suites/cross-cutting.xml

```diff
@@ -1,15 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
 <suite name="Tests for cross-cutting concerns" parallel="none">
-	<test name="Breadcrumb tests" preserve-order="true">
-		<classes>
-			<class name="gov.cancer.tests.crosscutting.BreadCrumb_Test" />
-		</classes>
-	</test>
-  <test name="Page title tests.">
+
+<test name="Breadcrumb tests" preserve-order="true">
+	<classes>
+	<!--	<class name="gov.cancer.tests.crosscutting.BreadCrumb_Test" />  -->
```

Comment:
You have removed the only test class specified in this group!

Reply:
fixed